.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Install dependencies
       run: |
         apt-get -y update
-        apt-get -y install libdbus-1-dev libssh2-1-dev parallel python3-dev python3-yaml python3-jsonschema python3-pip
+        apt-get -y install libdbus-1-dev libssh2-1-dev parallel python3-dev python3-yaml python3-jsonschema python3-pip golang-github-google-jsonnet
```

Comment:
in ubuntu you could try to install `jsonnet` package (the c++ version), it should be equivalent and useful for this check.
`golang-github-google-jsonnet` is the package name for the `go` version in openSUSE, that is why it cannot find it.

Reply:
updated